directdemod/merger/merger.py

```diff
@@ -0,0 +1,256 @@
+'''
+image merger
```

Comment:
Now that other two files are gone, there is no need for a separate folder. Please make a single merger.py in the main folder.

Reply:
Okay.